README.md

```diff
@@ -1,25 +1,91 @@
+| boost                      | 1.78            | A collection of C++ libraries                                                                                                       |
+| clang + llvm               | 16              | The LLVM compiler and toolchain for C/C++ and the libraries to interface with it                                                    |
+| cmake                      | 3.16.3          | A build system generator that creates makefiles used for compilation                                                                |
+| g++                        | 11              | A GNU compiler for C++                                                                                                              |
+| libssl-dev                 | 1.1.1           | Development package for OpenSSL, enabling various cryptography and secure network communication                                     |
+| libtool                    | 2.4.6           | Script that simplifies the process of creating and using shared libraries                                                           |
+| make                       | 4.3             | Reads makefiles generated by cmake to build executables from source code                                                            |
+| node                       | 16.17.0         | A JavaScript runtime                                                                                                                |
+| pkg-config                 | 0.29.1          | A helper tool used during compilation to provide correct compiler flags, and to locate and manage library dependencies              |
+| Rust toolchain             | 1.63            | Develop programs in Rust.<br>Use `cargo` to install `mdbook`, `mdbook-linkcheck`, `mdbook-mermaid`, and `sccache`                   |
+| Wasi SDK                   | 20.0            | A development toolchain for building WebAssembly programs that target the WebAssembly System Interface (WASI)                       |
+| zstd                       | 1.4.4           | Command-line utility that implements the Zstandard file compression algorithm                                                       |
+
+Other notes:
+* Add the paths to `node`, `npm`, `yarn`, `cargo`, and `wasm-opt` into your PATH environment variable.
+* Set the `WASI_SDK_PREFIX` environment variable before building to the root of your llvm installation (e.g. `/usr/lib/llvm-16`). Alternatively, set this variable during the build using the CMake flag `-DWASI_SDK_PREFIX=....`.
+* Add the wasm32-wasi target to rust.
+
+> 🔍 You may reference either the [Ubuntu 20.04 dockerfile](https://github.com/gofractally/image-builders/blob/main/docker/ubuntu-2004-builder.Dockerfile) or the [Ubuntu 22.04 dockerfile](https://github.com/gofractally/image-builders/blob/main/docker/ubuntu-2204-builder.Dockerfile) for an example on how an environment could be set up.
```

Comment:
"can" would be more appropriate than "may" here, as it is expressing capability rather than permission or uncertainty.

Reply:
Good point, updated (and also updated in a couple other places)